catalog/ui/catalog-ui-search/src/test/resources/forms/app/form-filter-simple.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?><Filter xmlns="http://www.opengis.net/fes/2.0" xmlns:ns2="http://www.opengis.net/ows/1.1" xmlns:ns3="http://www.w3.org/1999/xlink"><PropertyIsEqualTo><ValueReference>language</ValueReference><Literal>${value}</Literal></PropertyIsEqualTo></Filter>
```

Comment:
❓ I don't like this resource as a single line. I might look into pre-formatting it similar to how I'm doing it with JSON.